app/views/users/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Please Insert Your Log-in information</h2>
+
+<%# <h3>form with html</h3>
+<form action="/users" method="POST" accept-charset="UTF-8">
+       <label for="username">Username:</label>
+       <input type="text" id="username" name="user[username]"><br>
+       <label for="email">Email:</label>
+       <input type="text" id="email" name="user[username]"><br>
+       <label for="password">Password:</label>
+       <input type="text" id="password" name="user[username]"><br>
+      <%#  <<input type="submit" value="Submit">
+</form> %>
+
+ <%#  <h3>form with form_tags </h3>
+<%= form_tag("/users", method: "post") do %>
+    <%#   <%= label_tag(:username, "Username:") %>
+      <%#  <%= text_field_tag(:username) %>
+     <%#   <%= label_tag(:email, "Email:") %>
+     <%#  <%= text_field_tag(:email) %>
+     <%#  <%= label_tag(:password, "Password:") %>
+     <%#  <%= text_field_tag(:password) %>
+
```

Comment:
This is a good place to use the `password_filed_tag` since it was created for this specific purpose.